examples/dreamcast/micropython/Makefile

```diff
@@ -0,0 +1,37 @@
+TARGET = micropython.elf
+
+CFLAGS += -std=c99
+CFLAGS += -I.
+CFLAGS += -Wall -Og
+
+SRCS += example.c
+OBJS += $(SRCS:.c=.o)
```

Comment:
```suggestion
OBJS += example.o romdisk.o
```

Reply:
I didn't think he needed this line anymore with `KOS_ROMDISK_DIR=romdisk`?